src/PrestaShopBundle/Bridge/Smarty/SmartyBridge.php

```diff
@@ -125,7 +125,7 @@ public function render(
      *
      * @return string
      */
-    public function display(ControllerConfiguration $controllerConfiguration, string $content, $templateName): string
+    private function display(ControllerConfiguration $controllerConfiguration, string $content, $templateName): string
```

Comment:
Why make this private?

Reply:
Because this method is never used outside of this class, so no need to have an extra public endpoint when the `render` method is all we need